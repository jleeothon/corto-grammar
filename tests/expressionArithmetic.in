3 + 4 * a - 5 * (8 + 9)
