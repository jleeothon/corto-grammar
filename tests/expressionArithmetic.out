(program (statement (expression (commaExpression (additiveExpression (multiplicativeExpression (unaryExpression (postfixExpression (atomExpression (atom (literal 3)))))) + (multiplicativeExpression (unaryExpression (postfixExpression (atomExpression (atom (literal 4))))) * (unaryExpression (postfixExpression (atomExpression (atom (identifier a)))))) - (multiplicativeExpression (unaryExpression (postfixExpression (atomExpression (atom (literal 5))))) * (unaryExpression (postfixExpression (atomExpression ( (expression (commaExpression (additiveExpression (multiplicativeExpression (unaryExpression (postfixExpression (atomExpression (atom (literal 8)))))) + (multiplicativeExpression (unaryExpression (postfixExpression (atomExpression (atom (literal 9))))))))) )))))))) \n))
